src/components/HomePage/RenderHospital.tsx

```diff
@@ -3,11 +3,14 @@ import { Image, StyleSheet, Text, TouchableOpacity, View } from "react-native"
 import Stars from 'react-native-stars';
 import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
 import { stylesRenderHospital } from "../../styles/Home";
+import { useNavigation } from "@react-navigation/native";
+
 export const RenderHospital = ({ item }: any) => {
```

Comment:
Component shoul be Hospital instead of RenderHospital. It should be a noun instead of verb.